shardingsphere-distribution/shardingsphere-proxy-distribution/pom.xml

```diff
@@ -219,5 +226,199 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>native</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-compiler-plugin</artifactId>
+                        <version>${maven-compiler-plugin.version}</version>
+                        <configuration>
+                            <source>17</source>
+                            <target>17</target>
+                            <compilerArgument>--enable-preview</compilerArgument>
```

Comment:
I think turning on `--enable-preview` by default is a great choice, if someone wants to use GraalVM Native Image to test JDK19's Virtual Thread, they don't need to change much.